README.md

```diff
@@ -0,0 +1,20 @@
+# example-sentence-extraction
+Module: Finding example sentences for keywords
+This module is responsible for fetching sentences containing the target user-inputed keyword, and by comparison with one another, select no more than 5 sentences which demonstrate the appropriate usage of the keyword.
```

Comment:
Can we let users specify how many sentences to return as an argument? (i.e. we don't have to restrict the number of example sentences to just five)